SwiftGit2/Diffs.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  Diffs.swift
+//  SwiftGit2
+//
+//  Created by Jake Van Alstyne on 8/20/17.
+//  Copyright © 2017 GitHub, Inc. All rights reserved.
+//
+import libgit2
+
+public struct StatusEntry {
+	public var status: Diff.Status
+	public var headToIndex: Diff.Delta?
+	public var indexToWorkDir: Diff.Delta?
+
+	public init(from statusEntry: git_status_entry) {
+		self.status = Diff.Status(rawValue: statusEntry.status.rawValue)
+
+		if let htoi = statusEntry.head_to_index {
+			self.headToIndex = Diff.Delta(_: htoi.pointee)
+		}
+
+		if let itow = statusEntry.index_to_workdir {
+			self.indexToWorkDir = Diff.Delta(_: itow.pointee)
```

Comment:
You don't need to specify `_: ` here; I didn't even know you could do that. `Diff.Delta(itow.pointee)` is sufficient.

Reply:
nice catch